src/blocks/card.css

```
@@ -0,0 +1,84 @@
+.card {
+  box-sizing: border-box;
```

У тебя же это правило прописано в style.css))
```suggestion
  box-sizing: border-box;
```

---

Delete it from here